src/neo/Trie/MPT/MPTNode/ValueNode.cs

```diff
@@ -0,0 +1,37 @@
+using Neo.Cryptography;
+using Neo.IO;
+using System.IO;
+
+namespace Neo.Trie.MPT
+{
+    public class ValueNode : MPTNode
+    {
+        public byte[] Value;
+
+        protected override byte[] CalHash()
+        {
+            return Value.Length < 32 ? (byte[])Value.Clone() : Value.Sha256();
+        }
```

Comment:
?  The length of real value must less than 32?

Reply:
I was thinking if the value is short, just use the value as hash.

Reply:
Got, we can hiden this.